app/lending/(1-concepts)/collateralisation/page.mdx

```diff
@@ -0,0 +1,93 @@
+$$
+</div>
+
+A minimum collateralisation level is required (enforcing maximum LTV) when creating a loan. Borrowers can freely add more collateral at any time. If a user maintains an excessively low LTV (high overcollateralisation), a gradually increasing Low LTV Penalty is applied. This discourages locking up unnecessary collateral that reduces system-wide economic security.
+
+Chainflip supports multi-collateral accounts. The oracle value of all supported collateral assets is combined to determine total collateral value for LTV calculation.
+
+---
+
+### Oracle Safety Controls
+
+To protect borrowers and the system:
+
+- Liquidations halt if oracle prices become stale
+- New loan creation halts while oracles are stale
+- Borrowers may still repay, add collateral, or close positions during these pauses
+
+Future versions will support additional safety mechanisms like taking Chainflip's own DEX  prices into account.
+
+---
+
+### Auto-Collateralisation
+
+Borrowers may enable auto-top-up, which uses free balance in the State Chain account in their asset of choice to automatically increase collateral when LTV approaches the soft liquidation threshold. Auto-top-up prevents liquidation as long as the borrower has sufficient available balance. Note that for the Beta launch, Auto Top up will be disabled.
+
+---
+
+### Liquidation Overview
+
+Chainflip performs liquidations directly through its own cross-chain DEX. This provides two major advantages over traditional lending platforms:
+
+- Support for Patrial Liquidations: The protocol liquidates only what's necessary to bring LTV back to a safe range
+- Execution is native: LPs fill liquidation orders directly, removing the need for external liquidators demanding steep discounts
+
+Any LP on the Chainflip DEX can fill liquidation orders. There are no specialist liquidators.
+
+Liquidations begin when LTV crosses soft or hard liquidation thresholds.
+
+---
+
+### Soft Liquidation
+
+Soft liquidation is triggered when collateralisation drops below the soft threshold.
+
+The protocol submits small DCA-style liquidation orders with a minimum price slightly below the oracle value using Live Price Protection (LPP). LPs compete to fill these orders at near-market rates.
+
+Liquidation continues until:
+
+- the loan becomes healthy
+- the loan is fully repaid
+- collateral is exhausted
+
+Borrowers may also manually trigger soft liquidation to unwind a position and reclaim remaining collateral.
+
+---
+
+### Hard Liquidation
+
+Hard liquidation activates if LTV falls below the hard threshold.
+
+In this mode:
+
+- liquidation chunks are larger
+- oracle-indexed discount limits widen (up to 5%)
+- orders are submitted more aggressively
+
+Despite wider limits, LPs still compete on price, so execution generally remains close to market rates.
+
+The maximum expected liquidation loss for borrowers is approximately **5% of the loaned asset**, similar to Aave.
```

Comment:
Can we drop "expected"? I know we are trying to careful, but right now it seems to conflict with "maximum" and may look to someone like 5% loss is what they should always expect. I think we can reasonably argue that 5% is the max outright (the fact that the value of their crypto can drop due to global market conditions is true, but not intrinsic to our system).

```suggestion
The maximum liquidation loss for borrowers is **5% of the loaned asset**, similar to Aave.
```